.rhcicd/pr_check.sh

```diff
@@ -9,8 +9,10 @@ export IMAGE="quay.io/cloudservices/policies-ui-backend"
 export DEPLOY_TIMEOUT="600"
 
 # IQE plugin config
-export IQE_PLUGINS="policies"
-export IQE_MARKER_EXPRESSION="policies_api_smoke"
+#export IQE_PLUGINS="policies,floorist"
+export IQE_PLUGINS="floorist"
+#export IQE_MARKER_EXPRESSION="policies_api_smoke or floorist_smoke"
+export IQE_MARKER_EXPRESSION="floorist_smoke"
```

Comment:
This turns off (removes) policies testing.

Reply:
yes. I was just trying to troubleshoot. I was having issues with the image not having the plugin. I was just seeing what happened when I just tried to run the floorplan tests. This will not be in the final version. ALSO, Mike Shriver has shown me a new approach. It involves more updates to additional file that this, but will give us more flexibility.